specification/cognitiveservices/Language.Conversations/main.tsp

```diff
@@ -0,0 +1,1841 @@
+model NumberResolution extends BaseResolution {
+  @doc("A resolution for numeric entity instances.")
+  resolutionKind: ResolutionKind.NumberResolution;
+
+  @doc("The type of the extracted number entity.")
+  numberKind: NumberKind;
+
+  @doc("A numeric representation of what the extracted text denotes.")
+  value: float64;
+}
+
+@doc("A resolution for ordinal numbers entity instances.")
+model OrdinalResolution extends BaseResolution {
+  @doc("A resolution for ordinal numbers entity instances.")
+  resolutionKind: ResolutionKind.OrdinalResolution;
+
+  @doc("The offset With respect to the reference (e.g., offset = -1 in \"show me the second to last\"")
+  offset: string;
+
+  @doc("The reference point that the ordinal number denotes.")
+  relativeTo: RelativeTo;
+
+  @doc("A simple arithmetic expression that the ordinal denotes.")
+  value: string;
+}
+
+@doc("represents the resolution of a date and/or time span.")
+model TemporalSpanResolution extends BaseResolution {
+  @doc("represents the resolution of a date and/or time span.")
+  resolutionKind: ResolutionKind.TemporalSpanResolution;
+
+  @doc("represents the resolution of a date and/or time span. An extended ISO 8601 date/time representation as described in (https://github.com/Microsoft/Recognizers-Text/blob/master/Patterns/English/English-DateTime.yaml)")
+  begin?: string;
+
+  @doc("represents the resolution of a date and/or time span. An extended ISO 8601 date/time representation as described in (https://github.com/Microsoft/Recognizers-Text/blob/master/Patterns/English/English-DateTime.yaml)")
+  end?: string;
+
+  @doc("An optional duration value formatted based on the ISO 8601 (https://en.wikipedia.org/wiki/ISO_8601#Durations)")
+  duration?: string;
+
+  @doc("An optional modifier of a date/time instance.")
+  modifier?: TemporalModifier;
+
+  @doc("An optional triplet containing the beginning, the end, and the duration all stated as ISO 8601 formatted strings.")
+  @added(Versions.v2023_04_01)
+  timex?: string;
+}
+
+@doc("represents the resolution of numeric intervals.")
+model NumericRangeResolution extends BaseResolution {
+  @doc("represents the resolution of numeric intervals.")
+  resolutionKind: ResolutionKind.NumericRangeResolution;
+
+  @doc("The kind of range that the resolution object represents.")
+  rangeKind: RangeKind;
+
+  @doc("The beginning value of  the interval.")
+  minimum: float64;
+
+  @doc("The ending value of the interval.")
+  maximum: float64;
+}
+
+@doc("It is a wrap up of LUIS Generally Available response.")
+model LuisTargetIntentResult extends TargetIntentResult {
+  @doc("It is a wrap up of LUIS Generally Available response.")
+  targetProjectKind: targetProjectKind.Luis;
+
+  #suppress "@azure-tools/typespec-azure-core/bad-record-type" "LuisTargetIntentResult.result should not use ': Record<unknown>'. Use ': Record<string>' instead."
+  @doc("The actual response from a LUIS Generally Available application.")
+  result?: {};
```

Comment:
What is the expected service behavior here? Are we really returning an empty json object?

Reply:
This is not an empty object the result is defined as an object, and this is the intended behavior.

Reply:
In TypeSpec, this is an empty model. It has no properties. Can you provide an example of what the service will actually return?

Reply:
Do we know any part of the schema in what is returned? Or is it completely free-form?

Reply:
This seems weird but it is as per the swagger so we are not breaking anything here.

We will need someone from LUIS team to update the response here. Can we merge this and update in the next iteration?


```json
    "LuisTargetIntentResult": {
      "type": "object",
      "description": "It is a wrap up of LUIS Generally Available response.",
      "properties": {
        "result": {
          "type": "object",
          "description": "The actual response from a LUIS Generally Available application."
        }
      },
      "allOf": [
        {
          "$ref": "#/definitions/TargetIntentResult"
        }
      ],
      "x-ms-discriminator-value": "Luis"
    },

```

Reply:
I believe `{}` is ok to indicate an object type with an unknown schema. cc @markcowl

Reply:
If it is a json object, it would be a Record<unknown>. If it is arbitrary json (which includes e.g. a string or a number in addition to key/value pairs), it would be unknown.

Reply:
Based on Quentin's example...this does seem to lean more towards at least a Record<unknown> type

Reply:
Same question here, can we update this to at least be a `Record<unknown>` that way we're indicating to customers to expect from properties returned.

Reply:
I will update this to `Record<unknown>`. From the LUIS and CQA team, here is the explanation of why it is an open object? This result object is used to express responses from 3 different services - LUIS, QnA, CLU, It is declared as an object since the results from all services that use them will cast accordingly.

Reply:
@johanste  @catalinaperalta  can you take a look at the update.